### ISSUE TITLE
Initial notes

### ISSUE THREAD

**Issue by joshbruce** [4th-earth/site-fourth-earth#3]
Battle

tl;dr: #FourthEarth Lore seems more plausible every day. 

When it comes to Fourth Earth technology there are a few major constraints:

1. A lack of population necessary for mass production.
2. A lack of population for diversification of ideas.
3. Lack of population means loss of domain knowledge for teaching.
4. A what if question: what if access to firearms was restricted?

Sometimes a technology comes along that causes future innovation in a previous technology slows down or even stops (Steam Punk explores what might have happened if electric power didn’t overtake steam power, for example). In many countries firearms (guns firing bullets) are illegal. Hypotheses: Defense will always be pursued and defensive equalizers will tend to win.

In combat of any sort there are two major constraints: power and technique.

In unarmed combat the physically stronger will typically win regardless of technique (Bane versus the kid from Diary of a Wimpy Kid); however, a trained “wimpy kid” could overtake the physically stronger opponent (Fezzik vs. Westley).

To overcome this difference, add a knife. Then a sword. Both require less physical strength to be effective and require minimal training in technique (“More brawlers than warriors”). Add armor to defend - doesn’t require learning technique which takes time; shifting balance.

Add ranged weapons; hit them as much as you can before they get to you. Bows require the physical strength to overcome the pull of the bow and requires learning technique for aiming and firing; however, they have an advantage over throwing sticks as you can carry more arrows than you can throwing sticks. They become a battle of skill, part of which is the ability to reload and fire again before the target moves.

Enter firearms. The power input required for the system is the same for everyone due to mechanical advantage; if you can pull the trigger, you can fire the gun. The speed of the projectile also means you don’t have to adjust as much when aiming due to drop...less technique. The power output is also the same for everyone using the same type of bullet. If Tyson comes at me with what he’s good at and I have a forearm, it doesn’t matter, I can still win (Indiana Jones vs. The Saber guy). The great equalizer so to speak...minimal power input, minimal technique, standard output.

What if that is taken away and forces people to seek alternative methods of equalization??

My main bets when it comes to the world of Fourth Earth technology are:

1. Repurposing standard equipment and intentionally developing multipurpose tools; the battery and motor from this thing is the same as the battery and motors for that thing: I can use a spear as a walking stick.
2. Guerrilla tactics and mobility become inherent in tool development; silence, small, lightweight.
3. Minimal tooling required and an emphasis on reusability; bullets suck at this: I can make a suitable arrow out of a stick, pull it out and reuse it and my spear can continue to be used until broken (also made from a stick).

https://youtu.be/RNPpjuRtqeA
